python/mxnet/ndarray/numpy/_op.py

```diff
@@ -39,8 +39,7 @@
            'around', 'hypot', 'rad2deg', 'deg2rad', 'unique', 'lcm', 'tril', 'identity', 'take',
            'ldexp', 'vdot', 'inner', 'outer', 'equal', 'not_equal', 'greater', 'less', 'greater_equal', 'less_equal',
            'hsplit', 'rot90', 'einsum', 'true_divide', 'nonzero', 'shares_memory', 'may_share_memory', 'diff', 'resize',
-           'nan_to_num']
-
+           'nan_to_num', 'diagflat']
```

Comment:
move this to other places to avoid conflicts, probably you can move to after `full`.
Remember to move the function definitions too.

Reply:
Same for all other files.